src/CSET/operators/plot.py

```diff
@@ -230,7 +230,7 @@ def _plot_and_save_spatial_plot(
         The plotting method to use.
     """
     # Setup plot details, size, resolution, etc.
-    fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
```

Comment:
This is a good change, but should be done to all the plots. You can just to a search and replace of `figsize=(8, 8)` with `figsize=(10, 10)` in this file.